common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -13834,6 +13851,15 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=653)
     public static final String A_zimbraPrefPop3DownloadSince = "zimbraPrefPop3DownloadSince";
 
+    /**
+     * whether end user choose to use POP3 feature. If it is set to TRUE,
+     * POP3 feature is available only when zimbraPop3Enabled is TRUE.
+     *
+     * @since ZCS 9.0.0
```

Comment:
9.1.0 for all newly added attributes

Reply:
Next release version is yet to be finalised. Keeping 9.0.0